scripts/gen-deps-list.js

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env node
+'use strict'
+
+const { program, Argument } = require('commander')
+const DepTree = require('deptree')
+const fs = require('fs').promises
+const joinPath = require('path').join
+const semver = require('semver')
+const { getPackages } = require('./utils')
+const escapeRegExp = require('lodash').escapeRegExp
+
+const changelogConfig = {
+  path: joinPath(__dirname, '../CHANGELOG.unreleased.md'),
+  startTag: '<!--packages-start-->',
+  endTag: '<!--packages-end-->',
+}
+
+program
+  .argument('<package-name>', 'The name of the package to release')
+  .addArgument(new Argument('<release-type>', 'The type of release to perform').choices(['patch', 'minor', 'major']))
+  .option('-r, --read-changelog', 'Import existing packages from the changelog')
+  .option('-w, --write-changelog', 'Write output to the changelog')
+  .option('--force', 'Required when using --write-changelog without --read-changelog')
+  .showHelpAfterError(true)
+  .showSuggestionAfterError(true)
+  .parse()
+
+const [rootPackageName, rootReleaseType] = program.args
+const { readChangelog, writeChangelog, force } = program.opts()
+
+if (writeChangelog && !readChangelog && !force) {
+  program.showHelpAfterError(false).error(`
```

Comment:
Either add a comment explaining that the flow stops here or add a `throw` in front of it.